packages/localization/base.json

```diff
@@ -499,13 +499,20 @@
   "filterOffers.lowestAmount": "Lowest amount",
   "filterOffers.highestAmount": "Highest amount",
   "filterOffers.selectSortingMethod": "Select sorting method",
-  "filterOffers.searchByText": "Search by text",
```

Comment:
Create issue with no longer used localizations so we can remove those values later